table.csv

```diff
@@ -429,3 +429,8 @@ holochain-key-v0,               holochain,      0x947124,       Holochain v0 pub
 holochain-key-v1,               holochain,      0x957124,       Holochain v1 public key + 8 R-S (63 x Base-32)
 holochain-sig-v0,               holochain,      0xa27124,       Holochain v0 signature  + 8 R-S (63 x Base-32)
 holochain-sig-v1,               holochain,      0xa37124,       Holochain v1 signature  + 8 R-S (63 x Base-32)
+fil-piece-unsealed,             serialization,  0xf101,         Filecoin piece- raw data (CID = Piece Commitment)
+fil-sector-unsealed,            serialization,  0xf102,         Filecoin sector- raw data (CID = Data commitment)
+fil-sector-sealed,              serialization,  0xf103,         Filecoin sector- sealed and replicated (CID = Replication Commitment)
```

Comment:
Is the documentation available about the structure of all those ones? Are those really unique formats or just blobs of bytes? Multicodecs are for formats, not identifiers.

Reply:
@hannahhoward the piece commitment and the data commitment should be the same, only need one of those